api/frontgate/frontgate.proto

```diff
@@ -6,39 +6,88 @@ syntax = "proto3";
 
 package openpitrix.frontgate;
 
-service FrontgateService {
+service FrontgateServiceForPilot {
 	rpc GetInfo (Empty) returns (Info);
 	rpc CloseChannel (Empty) returns (Empty);
 
-	rpc StartConfd (Task) returns (Empty);
+	rpc GetConfdInfo(Empty) returns (ConfdInfo);
+	rpc StartConfd (StartConfdRequest) returns (Empty);
+	rpc StopConfd (StopConfdRequest) returns (Empty);
 
-	rpc RegisterMetadata (Task) returns (Empty);
-	rpc DeregisterMetadata (Task) returns (Empty);
+	rpc RegisterMetadata (RegisterMetadataRequest) returns (Empty);
+	rpc DeregisterMetadata (DeRegisterMetadataRequest) returns (Empty);
 
-	rpc RegisterCmd (Task) returns (Empty);
-	rpc DeregisterCmd (Task) returns (Empty);
+	rpc RegisterCmd (RegisterCmdRequest) returns (Empty);
+	rpc DeregisterCmd (DeRegisterCmdRequest) returns (Empty);
+}
+
+service FrontgateServiceForDrone {
+	rpc GetInfo (Empty) returns (Info);
```

Comment:
What is this rpc use for?  
Drone request to get frontgate info? May be no use.